Dockerfile

```diff
@@ -1,7 +1,5 @@
 # Compile
-FROM    rust:alpine3.16 AS compiler
-
-RUN     apk add -q --update-cache --no-cache build-base openssl-dev
+FROM    rust:bullseye AS compiler
```

Comment:
Unfortunately I cannot select a docker image from Rust with a pinned version of Debian

Let me know if I'm wrong: https://hub.docker.com/_/rust